tests/network/bgp/evpn/conftest.py

```
@@ -183,3 +194,38 @@ def vm_evpn_reference(
         vm.wait_for_agent_connected()
         yield vm
+
+
+@pytest.fixture(scope="module")
+def evpn_bridge(frr_external_pod: ExternalFrrPodInfo) -> None:
+    deploy_evpn_bridge(pod=frr_external_pod.pod)
+
+
+@pytest.fixture(scope="module")
+def external_l2_endpoint(
+    evpn_bridge: None,
```

Ruff (0.15.12)

[warning] Unused function argument: `evpn_bridge`

(ARG001)